tests/integration/test_package_managers.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import json
+import logging
+import os
+from pathlib import Path
+
+import pytest
+
+from . import utils
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "test_params",
+    [
+        # Test scratch build with parent from external registries
```

Comment:
This looks like copy-pasta from OSBS :smile:

Reply:
It is :blush: